### ISSUE TITLE
Example application?

### ISSUE THREAD

**Issue by matthiasdiener** [gtk-rs/sourceview#102]
Is there any example application showing to use sourceview?

**Comment by zer0x64** [comment]
I don't think this tutorial is official, but it includes an example of a sourceview use:
https://mmstick.github.io/gtkrs-tutorials/chapter_04/index.html

**Comment by gavr123456789** [comment]
@zer0x64 dead link
